test/blackbox-tests/test-cases/pkg/lock-directory-regeneration-safety.t/run.t

```diff
@@ -4,10 +4,16 @@ Create a lock directory that didn't originally exist
   > (lang dune 3.10)
   > (lock_dir
   >  (repositories mock))
+  > (lock_dir
+  >  (path "dev/dune.lock")
+  >  (repositories mock))
   > (repository
   >  (name mock)
   >  (source "file://$(pwd)/mock-opam-repository"))
   > EOF
+  $ dune pkg lock "dev/dune.lock"
+  Solution for dev/dune.lock:
+  (no dependencies to lock)
```

Comment:
Did this test not fail? Or did you just forget to promote it?

Reply:
My bad: I've mixed the failing test with the promote one... I'll split them in two :+1: